sentences/nl/homeassistant_HassTurnOn.yaml

```diff
@@ -18,14 +20,11 @@ intents:
           device_class: garage
           domain: cover
       - sentences:
-          - open [de | het](gordijn | gordijnen | luxaflex | rolluik | rolluiken |
-            screen | screens) in <area>
+          - open [de | het](gordijn | gordijnen | luxaflex | rolluik | rolluiken | screen | screens) in <area>
```

Comment:
Just a typo in the sentence: 
- open [de | het](gordijn | gordijnen | luxaflex | rolluik | rolluiken | screen | screens) in <area>
- open [de | het] (gordijn | gordijnen | luxaflex | rolluik | rolluiken | screen | screens) in <area>

Reply:
Fixed